exporter/kafkaexporter/README.md

```diff
@@ -9,11 +9,13 @@ The following settings are required:
 
 The following settings can be optionally configured:
 - `brokers` (default = localhost:9092): The list of kafka brokers
-- `topic` (default = otlp_spans): The name of the kafka topic to export to
-- `encoding` (default = otlp_proto): The encoding of the payload sent to kafka. Available encodings:
-  - `otlp_proto`: the payload is serialized to `ExportTraceServiceRequest`.
+- `topic` (default = otlp): The name of the kafka topic to export to
+- `traces_encoding` (default = otlp_proto): The encoding of the traces sent to kafka. All available traces encodings:
```

Comment:
This is a breaking change. Is it needed? What's the use case when traces and metrics need to use different encodings? Even if there is a such use case one can have 2 different exporters configured with different encodings in that case.

Can we keep the `encoding` key? Obviously settings that only work for traces will be invalid for a metric exporter (e.g. `jaeger_proto` should not be used for an exporter attached to metric pipeline).

Reply:
Right, this *is* a breaking change, potential confusion between valid encoding based on metrics/traces is why it was split up this way. I will revert this to use only the `encoding` key and update the instructions to indicate which encoding can be used for which context